test/utils/factories/user.js

```diff
@@ -8,10 +8,10 @@ const stubs = require('../../utils/stubs');
 
 const chance = new Chance();
 
-module.exports.getValidUser = function getValidUser() {
+module.exports.getUser = function getUser(phoneNumber) {
```

Comment:
I didn't have time to implement it in blink, but I'd recommend looking into [`factory_girl`](https://github.com/aexmachina/factory-girl). It may be more convenient, as it allows both to generate fields from chance.js and override them on demand (as your're doing here with a phoneNumber).

Reply:
I'll take a look in future integration tests.

Reply:
Yep, just a note.